src/controllers/queries/index.ts

```diff
@@ -352,31 +352,39 @@ export class QueriesController extends HTMLController {
       throw new InvalidInputError(`Missing response_id to respond to.`)
     }
 
-    const partialConnections: PartialQuery[] = []
     if (partial && partialQuery) {
       if (!partial.connectionIds || !partial.productIds || !partial.quantities) {
         throw new InvalidInputError('missing a property in the request body')
       }
       req.log.info('processing partial query %j', partial)
       const size: number = this.validatePartialQuery(partial)
-      req.log.debug('partial query has been validated %j', { partial, size })
+      req.log.debug('validate partial query and submitting DRPC request %j', { partial, size })
 
       for (let i = 0; i < size; i++) {
```

Comment:
Maybe consider doing this in a `Promise.all` rather than serially?

Reply:
updated to use Promise.all